dockerfile_scripts/additional-requirements-rocm.txt

```diff
@@ -10,7 +10,7 @@ Pillow>=8.3.2,<=9.5.0
 analytics-python
 nvidia-ml-py
 protobuf<=3.20.3
-tensorboard==2.10.1
+tensorboard==1.15
```

Comment:
hold up, this can't be right, this is a tensorboard version for tensorflow v1 from 2019.
can we have a version that is more modern?

Reply:
We can test a different version but 2.10.1 fails on end-to-end testing and was imortalized in jira PAD-91:

Reply:
I have a strong opinion we cannot pin 1.15 here because it's 5 years old and likely to conflict with other dependencies and have CVEs.
2.10.1 is also technically above 1.15...